docs/docs/refreshing-content.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Refreshing Content
+---
+
+During local development it can be useful to refresh sourced content without restarting the development server. To faciliate this Gatsby exposes an environmental variable `ENABLE_GATSBY_REFRESH_ENDPOINT`.
```

Comment:
```suggestion
During local development, it can be useful to refresh sourced content without restarting the development server. To facilitate this, Gatsby exposes an environment variable called `ENABLE_GATSBY_REFRESH_ENDPOINT`.
```